sormas-ui/src/main/java/de/symeda/sormas/ui/user/UserController.java

```diff
@@ -77,6 +78,16 @@ public void create() {
 	public void edit(UserDto user) {
 		Window window = VaadinUiUtil.createPopupWindow();
 		CommitDiscardWrapperComponent<UserEditForm> userComponent = getUserEditComponent(user.getUuid(), window::close);
+
+		if (FacadeProvider.getFeatureConfigurationFacade().isFeatureEnabled(FeatureType.KEYCLOAK_TO_SORMAS_USER_SYNC)) {
```

Comment:
This should be done in the user edit form and the feature check should be done on the backend too to disallow user edit/delete via the rest api

Reply:
made the changes